rust-connectors/sinks/sql/README.md

```diff
@@ -0,0 +1,98 @@
+| Model           | PostgreSQL                   | SQLite       |                                          
+|:----------------|:-----------------------------|:-------------|
+| Bool            | BOOL                         | BOOLEAN      |
+| Char            | CHAR                         | INTEGER      |
+| SmallInt        | SMALLINT, SMALLSERIAL, INT2  | INTEGER      |
+| Int             | INT, SERIAL, INT4            | INTEGER      |
+| BigInt          | BIGINT, BIGSERIAL, INT8      | BIGINT, INT8 |
+| Float           | REAL, FLOAT4                 | REAL         |
+| DoublePrecision | DOUBLE PRECISION, FLOAT8     | REAL         |
+| Text            | VARCHAR, CHAR(N), TEXT, NAME | TEXT         |
+| Bytes           | BYTEA                        | BLOB         |
+| Numeric         | NUMERIC                      | REAL         |
+| Timestamp       | TIMESTAMP                    | DATETIME     |
+| Date            | DATE                         | DATE         |
+| Time            | TIME                         | TIME         |
+| Uuid            | UUID                         | BLOB, TEXT   |
+| Json            | JSON, JSONB                  | TEXT         |
+
+
+## Transformations
+The SQL Sink connector expects the data in [Fluvio SQL Model](../../models/fluvio-model-sql) in JSON format.
+In order to work with different data formats or data structures, transformations can be applied.
+The transformation is a SmartModule pulled from the SmartModule Hub. Transformations are chained according to the order
+in the config. If a SmartModule requires configuration, it is passed via `with` section of `transforms` entry. 
+## Usage
+
+### Configuration via parameters
+| Option       | default                     | type   | description                                           |
+|:-------------|:----------------------------| :---   |:------------------------------------------------------|
+| hub_url      | http://127.0.0.1:8080       | String | The paramaters key for the smartmodule hub url        |
+| database_url | -                           | String | The paramaters key for the sql database conection url |
+
+
+### Configuration via secrets
+| Option              | default                     | type   | description                                           |
+|:--------------------|:----------------------------| :---   |:------------------------------------------------------|
+| FLUVIO_HUB_URL      | http://127.0.0.1:8080       | String | The secret key for the smartmodule hub url        |
+| FLUVIO_DATABASE_URL | -                           | String | The secret key for the sql database conection url |
+
+### Example
+Let's look at the example of the connector with one transformation named `infinyon/json-sql`. The transformation takes
+records in JSON format and creates SQL insert operation to `topic_message` table. The value from `device.device_id`
+JSON field will be put to `device_id` column and the entire json body to`record` column.
+
```

Comment:
Please provide the `JSON` object so the reader can rationalize the hierarchy.

Reply:
Done